fun foo(a) {
  let a; #!! [line 2] resolve error at 'a': variable with this name already declared in this scope
}
